frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/views/PipelineEdit.java

```diff
@@ -908,7 +932,9 @@ public void textChange(FieldEvents.TextChangeEvent event) {
         pipelineSettingsLayout.addComponent(pipelineDescription, 1, 1);
 
         Label visibilityLabel = new Label(Messages.getString("PipelineEdit.visibility"));
-        pipelineSettingsLayout.addComponent(visibilityLabel, 0, 2);
+        if (!"organization".equals(appConfig.getString(ConfigProperty.OWNERSHIP_TYPE))) {
+            pipelineSettingsLayout.addComponent(visibilityLabel, 0, 2);
```

Comment:
there was request for edem to hide selection of visibility modes to the
user, later the request was updated to hide it only for non-admin user, so
i changed the code according to later requirement

On Sun, Mar 22, 2015 at 12:50 PM, Tomas Knap <notifications@github.com>
wrote:

> In
> frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/views/PipelineEdit.java
> <https://github.com/UnifiedViews/Core/pull/282#discussion_r26902105>:
>
> > @@ -908,7 +932,9 @@ public void textChange(FieldEvents.TextChangeEvent event) {
> >          pipelineSettingsLayout.addComponent(pipelineDescription, 1, 1);
> >
> >          Label visibilityLabel = new Label(Messages.getString("PipelineEdit.visibility"));
> > -        pipelineSettingsLayout.addComponent(visibilityLabel, 0, 2);
> > +        if (!"organization".equals(appConfig.getString(ConfigProperty.OWNERSHIP_TYPE))) {
> > +            pipelineSettingsLayout.addComponent(visibilityLabel, 0, 2);
>
> What is the purpose of this line? Why is the visibilityLabel hidden for
> organization mode?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UnifiedViews/Core/pull/282/files#r26902105>.
>